lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -140,7 +140,7 @@ def indent(count, skip1: false)
           apipie :method, 'Performs a DNS lookup on Foreman server' do
             required :name_or_ip, String, desc: 'a hostname or IP address to perform DNS lookup for'
             returns String, desc: 'IP resolved via DNS if hostname was given, hostname if an IP was given.'
-            raises error: Timeout::Error, desc: 'when DNS resolve could not be performed in time set by global setting `dns_confict__timeout`'
+            raises error: Timeout::Error, desc: 'when DNS resolve could not be performed in time set by global setting `dns_timeout`'
```

Comment:
This one I'm not 100% sure about. `confict__` looks like something generated by script, but is only instance in foreman source.
`confict` does sound similar to `conflict`, but there are no references to DNS conflict whatsoever.

Meanwhile, `dns_timeout` is referenced just couple of lines below, so I'm assuming this is what was meant here.

Reply:
correct, the setting is `dns_timeout`